chapters/5-processing.tex

```diff
@@ -100,32 +107,37 @@ \subsection{Unique identifiers}
   reflecting only that information'',
   depending on the context.
 	This applies to data acquired from partners as well as
-	primary data collected by the research team.}
-It is also possible for a raw dataset to not include a unique identifier,
-or for the identifier to involve a long string
-that is difficult to work with, such as a person's name.
-In such cases, cleaning begins by either carefully creating a variable
-that uniquely identifies the data,
-or merging the ID variable from the master dataset
+	original data collected by the research team.
+}
+
+It is also possible for a raw dataset to not include an unique identifier,
+or that the identifier is not a suitable \textbf{project ID}.\sidenote{
+	\textbf{Project ID:} The main ID used in your project to identify
+	observations.
+	You should never have multiple project IDs for the same unit of observation.
+	The project ID must uniquely and fully identify all observations in the project.
+	See \url{https://dimewiki.worldbank.org/ID\_Variable\_Properties} for more details.}
+Suitable project IDs should, for example, not involve long strings
+that are difficult to work with, such as a person's name,
+or be an ID that is known outside the research team.
```

Comment:
```suggestion
or be an ID that is used outside the research team.
```

Reply:
I think _known_ is the correct word here. Relating to the discussion we just had on Slack, the key is that no-one knows the ID outside the research team. It is an direct extension of the recommendation that only people listed on the IRB should have access to the identified data. If we want to, we can say that. That "the link between study ID and PII should only be known by people listed on the IRB".

And then I think when we publish the data, for example in a replication package, we can include the study ID, and other researchers can use it to combine data in that replication package in new ways. that usage of the study ID outside the research team is fine, as long as there is no way for them to know how it links to PII.

Reply:
In chapter 3 we already say:

> The project ID must be created by the project team, and the linkage to direct identifiers should only be known to people listed on the IRB.

Reply:
@mariaruth and @bbdaniels , I updated Maria's original suggestion. Let me know what you think about this.

Reply:
Yes, I like this, and I would change "person's name" on 121 to just "name", since "person's name" is _obviously_ identifying and should almost never be used at all

Reply:
> Yes, I like this, and I would change "person's name" on 121 to just "name", since "person's name" is _obviously_ identifying and should almost never be used at all

done in 9ee3119c5ff4ed94f2b383a1ef6201418736cbb3